### ISSUE TITLE
about test

### ISSUE THREAD

**Issue by Jeling-W** [icoz69/CEC-CVPR2021#5]
First of all thank you for your contribution.
It is mentioned in the paper that in each incoming session, we use the adaptation module to update the classifiers learned in the current session and previous sessions, and then concatenate the updated classifiers to make predictions over all classes. How to predict the categories that have not appeared in the base data and the  previous and current session? Why the test accuracy gradually decrease with the increase of the session?

**Comment by icoz69** [comment]
hi, thanks for the interest in our work. The eval base session is not important in this task, as it's nothing about few-shot and incremental learning. You can directly use the pre-trained CNN to eval the base classes. Alternatively, you can also apply the graph model to the classifier learned in the base session.
As to your second question, it is because more classes are involved and thus and classification task is becoming more challenging.

**Comment by Jay-IPL** [comment]
Hi thanks for the nice work!
In the paper it says, 'in each incoming session, we use the adaptation module to update the classifiers learned in the current session and previous sessions, and then concatenate the updated classifiers to make predictions over all classes.' For example, given one input image, the based classifier will give us probabilities over base classes, then the 2nd classifier will also give us probabilities over its own classes different from base classes, so does the other classifiers. With all these probabilities over different classes, which one should the model choose to be the final prediction? the highest probability?
